demos/imagenet_classification_demo/include/my_functor.hpp

```diff
@@ -0,0 +1,79 @@
+#include <cstdio>
+#include <string>
+#include <functional>
+#include <atomic>
+
+#include <inference_engine.hpp>
+
+#include <ie_iextension.h>
+#include <ext_list.hpp>
```

Comment:
```suggestion

```
This include also appears in `main.cpp` and should also be removed.

Reply:
Done